templates/zerver/emails/custom_email_base.pre.manage_preferences_block.txt

```diff
@@ -0,0 +1,15 @@
+
+---
+{% if remote_server_email %}
+You are receiving this email to update you about important changes to Zulip's Terms of Service.
+
+Unsubscribe from marketing emails: {{ unsubscribe_link }}
```

Comment:
Should this one be tagged for translation like the one just below?

Also, I think the `:` maybe belongs outside the translation tag, if we want to match the existing `templates/zerver/emails/custom_email_base.pre.html`

Reply:
> Should this one be tagged for translation like the one just below?

https://github.com/zulip/zulip/pull/31534#discussion_r1746622637

> Also, I think the : maybe belongs outside the translation tag, if we want to match the existing templates/zerver/emails/custom_email_base.pre.html

That's correct, fixed that. We make similar mistakes in `digest.txt` and `missed_message.txt` so I'll fix that in a follow-up PR. That'll help in avoiding effort to translate the same thing with/without ":" in those languages for which it's not translated yet.